public/icons/mask-icon.svg

```diff
@@ -3,16 +3,10 @@
     <!-- Generator: Sketch 47.1 (45422) - http://www.bohemiancoding.com/sketch -->
```

Comment:
Unless this is required by the license, could you remove all the unnecessary fields to reduce file size?